src/Network/Network/AzureFirewallPolicy/NewAzureFirewallPolicyCommand.cs

```diff
@@ -130,14 +130,25 @@ public class NewAzureFirewallPolicyCommand : AzureFirewallPolicyBaseCmdlet
             HelpMessage = "ResourceId of the user assigned identity to be assigned to Firewall Policy.")]
         [ValidateNotNullOrEmpty]
         [Alias("UserAssignedIdentity")]
-        public string UserAssignedIdentityId { get; set; }
+        public string[] UserAssignedIdentityId { get; set; }
 
         [Parameter(
             Mandatory = false,
             HelpMessage = "Firewall Policy Identity to be assigned to Firewall Policy.")]
         [ValidateNotNullOrEmpty]
         public PSManagedServiceIdentity Identity { get; set; }
 
+        [Parameter(
```

Comment:
To remove the a parameter is a breaking change, which can only be release in the major release. The next one is in November.